### ISSUE TITLE
Please add www.simplywhisked.com

### ISSUE THREAD

**Issue by duckunix** [hhursev/recipe-scrapers#255]
Sample URL: https://www.simplywhisked.com/farfalle-with-italian-sausage-and-broccoli/#tasty-recipes-16991

I used the FAQ to verify the scraper works:
```
14:21:32 [don@harper:~] $ python
Python 3.8.6 (default, Sep 25 2020, 09:36:53) 
[GCC 10.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from recipe_scrapers import scrape_me
>>> scraper = scrape_me('https://www.simplywhisked.com/farfalle-with-italian-sausage-and-broccoli/#tasty-recipes-16991', wild_mode=True)
>>> scraper.title()
'Italian Sausage Pasta with Broccoli'
>>>
```

**Comment by nhomka** [comment]
Added in Pull Request #257 

First contribution to the project, the schema was all compatible so there was really nothing to it, let me know if something isn't covered here.

**Comment by duckunix** [comment]
Awesome!  Thanks.  I got a little distracted from trying to figure out what to do, so I gave up.  Thanks for a nice clean diff I can study.  I have some more sites I want to add, and this way, I can do it myself!

**Comment by bfcarpio** [comment]
closed by #257

**Comment by duckunix** [comment]
Tested and works great!
